### PR TITLE
feat(cmd): pactus-wallet add info commands

### PR DESCRIPTION
**Pull request by maxipaz**
## Description

Adding `info` commands for `pactus-wallet`

## Related issue(s)

- https://github.com/pactus-project/pactus/issues/1494

### CODE REVIEW

File: wallet/wallet.go [hunk 1]
```diff
@@ -502,3 +502,15 @@ func (w *Wallet) SignMessage(password, addr, msg string) (string, error) {
 
 	return prv.Sign([]byte(msg)).String(), nil
 }
+
+func (w *Wallet) GetVersion() int {
```
Comment (b00f) [inline wallet/wallet.go:506]: Just a note: In our codebase, we typically avoid using `Get` as a prefix for getter methods. Let's keep our naming consistent with this convention.

File: cmd/wallet/main.go [hunk 2]
```diff
@@ -63,6 +63,7 @@ func main() {
 	buildAllTransactionCmd(rootCmd)
 	buildAllAddrCmd(rootCmd)
 	buildAllHistoryCmd(rootCmd)
+	buildAllInfoCmd(rootCmd)
```
Comment (b00f) [inline cmd/wallet/main.go:66]:
```suggestion
	buildInfoCmd(rootCmd)
```

File: cmd/wallet/info.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+// buildAllInfoCmd builds all sub-commands related to the wallet information.
+func buildAllInfoCmd(parentCmd *cobra.Command) {
+	infoCmd := &cobra.Command{
+		Use:   "info",
+		Short: "retrieving the wallet information.",
+	}
+
+	parentCmd.AddCommand(infoCmd)
+	buildWalletVersionInfoCmd(infoCmd)
```
Comment (b00f) [inline cmd/wallet/info.go:18]: I don't think we need subcommands here. It would be more convenient for the user to get all the wallet information at once.

**Review by b00f — CHANGES_REQUESTED** [review]

File: wallet/wallet.go (hunk 1 above)
Comment (maxipaz) [reply wallet/wallet.go]: Fixed. I also renamed `GetHistory` function that was a bit below this function

File: cmd/wallet/info.go (hunk 3 above)
Comment (maxipaz) [reply cmd/wallet/info.go]: Done 🙂

**Review by b00f — APPROVED** [review]

File: wallet/wallet.go (hunk 1 above)
Comment (b00f) [reply wallet/wallet.go]: Thanks

**Pull request merged**